guides/v2.0/install-gde/prereq/dev_install.md

```diff
@@ -43,14 +43,11 @@ This section discusses how to get current code by cloning the Magento GitHub's d
 
 Currently, the `develop` branch is the default but you can checkout a release branch like `2.1.0` after cloning.
 
-### Creating an authorization file {#instgde-prereq-compose-clone-auth}
+## Create a Composer authentication file {#instgde-prereq-compose-clone-auth}
```

Comment:
We no longer update 2.0 files. You must make all these changes in the 2.1 file instead, which is symlinked to 2.2 and 2.3.

Reply:
No need to revert changes from 2.0. In fact, I agree with you that it's a good idea to go ahead and update it since it uses the same include.